torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1044,14 +1044,16 @@ def call_function(
         args: List[VariableTracker],
         kwargs: Dict[str, VariableTracker],
     ) -> VariableTracker:
+        from torch._higher_order_ops.associative_scan import first_slice_copy
+
         from .builder import wrap_fx_proxy
 
         args, kwargs = LazyVariableTracker.realize_all((args, kwargs))
 
-        def arg_extractor(combine_fn, xs, dim):
-            return combine_fn, xs, dim
+        def arg_extractor(combine_fn, xs):
```

Comment:
i feel we should split this diff into 2, first is the the dim change, then the autograd.